.github/workflows/build-starter-templates.yml

```diff
@@ -0,0 +1,55 @@
+name: Build TinaCMS Starter Templates
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 9 * * 1-5'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package-manager: [yarn, npm, pnpm]
+        template:
+          - tina-cloud-starter
+          - tina-hugo-starter
+          - tina-remix-starter
+          - demo-docs
+          - tinasaurus
+          - basic
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22
```

Comment:
nitpick - should we use v4?

Reply:
use v4 for checkout + setup-node
also node version should be a matrix parameter.... need to support 18, 20, 22

Reply:
Done 
- Updated to v4
- Added `node-version` param for version 18, 20, and 22